docs/explanations/pro_token_and_machine_usage.rst

```diff
@@ -0,0 +1,49 @@
+.. _pro_token_and_machine_usage:
+
+Machine counts and token usage for Ubuntu Pro
+*********************************************
+
+Ubuntu Pro tokens are used to authenticate your machine and enable access to the services you are entitled to. These tokens are unique and specific to your account, so you should treat them as a secret (as you would with a password).
+
+In this page, we'll discuss how Ubuntu Pro tokens work and how machines are counted against your subscription.
+
+Ubuntu Pro token(s)
+--------------------
+
+- Obtaining Pro tokens:
+     After subscribing to Ubuntu Pro, you can access your subscriptions through the `Ubuntu Pro Dashboard <Pro_dashboard_>`_. Each subscription includes a Ubuntu Pro token that can be used to attach as many machines as the subscription allows.
+
+- Attaching Pro tokens:
+    Command: Use ``sudo pro attach`` to connect a machine to your Ubuntu Pro subscription.
+
+- Detaching Pro tokens:
+    Command: Use ``sudo pro detach`` to remove a machine from the subscription. If you are attached on a VM you intend to delete, the token does not need to be detached before the VM is destroyed.
+
+For further information regarding pro commands, refer to our :ref:`Getting started-with Ubuntu Pro client tutorial<tutorial-commands>`.
+
+Monitoring active machines:
+---------------------------
+
+Active machines are counted based on the number of machines attached to your subscription at a given time. An active machine is defined as a machine that has been attached to a given subscription at any point within the past 24 hours.
+
+Each subscription type has a limit on the number of machines it can cover. For personal subscriptions, this limit is 5 physical machines with unlimited VMs on these machines. Enterprise subscriptions will have a higher limit and may differentiate between physical and virtual machines.
```

Comment:
I wonder if these three options would look a bit clearer if presented as bullet points?

Reply:
I think the bullet points here break the continuity, currently all three points look better as paragraphs conveying different points about Active machines and related information. Dividing them into bullets would force some topic on each point which could be more confusing in my opinion.

Reply:
That's fair - feel free to disregard this suggestion then